tensorflow/python/compiler/tensorrt/utils.py

```diff
@@ -257,3 +261,79 @@ def draw_graphdef_as_graphviz(graphdef, dot_output_filename):
   print("  - `sudo apt install -y graphviz`")
   print("  - `dot -Tpng <input_filename>.dot -o <output_filename>.png`")
   print("===================================================================\n")
+
+
+@functools.lru_cache
+def get_nvidia_gpu_desc_dict():
```

Comment:
This routine is not used outside this module, right?
Let's make it internal in this case.